source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -106,9 +110,9 @@ Some environments do have some additional requirements / checks that need to be
 We upgrade PTL/PTLSbox and Preview/Dev in-place, meaning we aren't building another cluster or switching in the appgw unless necessary. If one of these environments must instead be rebuilt and swapped over, please refer to the Rebuilding AKS Clusters guide.
 
 ### Ptlsbox, Ptl
-Make an announcement that Jenkins may be briefly unavailable in the cloud-native-announce slack channel. 
-This environment is best to be done early in the morning.
-See [Jenkins instances](/jenkins/) to check they are running after upgrades.
+Make an announcement that Jenkins may be briefly unavailable in the #cloud-native-announce slack channel for the CFT cluster. 
+Alternatively, for the SDS cluster, kindly utilize the #sds-cloud-native channel. It is recommended to issue this announcement early in the morning. 
```

Comment:
duplication of above

Reply:
Removed and made the change to the above